rpc/block_svc_routines.c

```diff
@@ -4362,6 +4362,8 @@ block_create_common(blockCreate *blk, char *control, char *volServer, char *prio
   blockServerDefPtr list = NULL;
   size_t i;
   bool prioCap = false;
+  char *out = NULL;
+  int ret = true;
```

Comment:
Where are we using these ?

Reply:
Yeah, good catch.
I meant to use the "out" instead of "save" in the followed code. But hit one odd issue which I have fixed in new commit followed, forgot to remove this.

Will fix it. thanks.